PhysicsTools/NanoAOD/python/jets_cff.py

```diff
@@ -531,6 +531,66 @@
 run2_jme_2017.toModify( cjetNN,outputFormulas = cms.vstring(["at(0)*0.24718524515628815+0.9927206635475159","0.5*(at(2)-at(1))*0.24718524515628815"]))
 
 
+#
+# ML-based FastSim refinement
+#
+from Configuration.Eras.Modifier_fastSim_cff import fastSim
+def nanoAOD_refineFastSim_bTagDeepFlav(process):
```

Comment:
I'm not really thrilled with making a function in this file, can you make it elsewhere and then simply import it here?

Reply:
For now there is only a refinement for AK4 CHS... Perhaps, since this was done the same way in the original PR to master, we could leave it that way for now, and when additional refinements are added for AK4 Puppi/AK8/..., they could be regrouped into a single file, e.g. `fastsim_refinements_cff.py`?